### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by efim-poberezkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,54 +124,66 @@ $ stack install
...
+Once you've set up your local profile, use `/connect` to create a new connection and generate an invitation. The `connect` command will output an invitation. Send this invitation to your contact via any other channel.
 
 The invitation has the format `smp::<server>::<queue_id>::<rsa_public_key_for_this_queue_only>`. The invitation can only be used once and even if this is intercepted, the attacker would not be able to use it to send you the messages via this queue once your contact confirms that the connection is established.
 
-The contact who received the invitation should use `/connect <name2> <invitation>` to accept the connection (`<name2>` is any name that the accepting contact wants to use for you).
-
-Once the contact has used the `/connect` command, a connection is established and both parties are notified.
+The contact who received the invitation should use `/connect <invitation>` to accept the connection. This establishes the connection, and both parties are notified.
 
 They would then use `@<name> <message>` commands to send messages. One may also press Space or just start typing a message to send a message to the contact that was the last.
```
[the remark this answers is not in the document]
Comment (efim-poberezkin) [reply README.md:171]: I think yes

File: README.md [hunk 2]
```diff
@@ -52,17 +56,17 @@ The routing of messages relies on the knowledge of client devices how user conta
 
 ## Current features of the terminal chat
 
-- 1-to-1 chat with multiple people in the same terminal window.
+- Direct and group messaging and file transfer.
```
Comment (epoberezkin) [inline README.md:59]: I'd split in 3 points

File: README.md [hunk 3]
```diff
@@ -121,54 +125,68 @@ $ stack install
 
 ### Running the chat client
 
-To start the chat client, run `dog-food` (as in [eating your own dog food](https://en.wikipedia.org/wiki/Eating_your_own_dog_food)) from the terminal.
-
-By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and SQLite database file `smp-chat.db` is initialized in it.
+To start the chat client, run `simplex-chat` from the terminal.
 
-The default SMP server is `smp1.simplex.im#pLdiGvm0jD1CMblnov6Edd/391OrYsShw+RgdfR0ChA=` (base-64 encoded string after server host is the transport key digest) - it is pre-configured in the app.
+By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and two SQLite database files `simplex.chat.db` and `simplex.agent.db` - for chat client and agent respectively - are initialized in it.
```
Comment (epoberezkin) [inline README.md:130]: "for chat client and agent respectively" - not needed

File: README.md [hunk 4]
```diff
@@ -121,54 +125,68 @@ $ stack install
...
+To specify a different file path prefix for the database files use `-d` command line option:
 
 ```shell
-$ dog-food -d my-chat.db
+$ simplex-chat -d alice
 ```
 
+Running above, for example, would create `alice.chat.db` and `alice.agent.db` database files in current directory.
+
+Default SMP servers are `smp2.simplex.im#z5W2QLQ1Br3Yd6CoWg7bIq1bHdwK7Y8bEiEXBs/WfAg=` (UK) and `smp3.simplex.im#nxc7HnrnM8dOKgkMp008ub/9o9LXJlxlMrMpR+mfMQw=` (California) - they are pre-configured in the app. Base-64 encoded string after server host is the transport key digest.
```
Comment (epoberezkin) [inline README.md:140]: are hosted on Linode ... London, UK / Freemont, CA. No need to put the addresses here, you can link to the source code.

File: README.md [hunk 5]
```diff
@@ -121,54 +125,68 @@ $ stack install
...
-$ dog-food -s smp.example.com:5223#KXNE1m2E1m0lm92WGKet9CL6+lO742Vy5G6nsrkvgs8=
+$ simplex-chat -s smp.example.com:5223#KXNE1m2E1m0lm92WGKet9CL6+lO742Vy5G6nsrkvgs8=
 ```
 
 The base-64 encoded string in server address is the digest of RSA transport handshake key that the server will generate on the first run and output its digest.
 
 You can still talk to people using default or any other server - it only affects the location of the message queue when you initiate the connection (and the reply queue can be on another server, as set by the other party's client).
 
-Run `dog-food --help` to see all available options.
+Run `simplex-chat --help` to see all available options.
```
Comment (epoberezkin) [inline README.md:152]: -h

File: README.md [hunk 6]
```diff
@@ -121,54 +125,68 @@ $ stack install
...
 
 ### How to use SimpleX chat
 
+Once you have started the chat, you will be prompted to specify your "display name" and an optional "full name" as part of creation of your local user profile. Your display name is an alias for your contacts to refer to you by - it is not unique and does not serve as a global identity. In case different contacts chose the same display name, the chat client adds a numeric suffix to their local display names.
+
 This diagram shows how to connect and message a contact:
 
 <div align="center">
-  <img align="center" src="images/how-to-use-simplex.svg">
+  <img align="center" src="images/how-to-use-simplex.png">
```
Comment (epoberezkin) [inline README.md:161]: can't it be svg?

**Pull request merged**